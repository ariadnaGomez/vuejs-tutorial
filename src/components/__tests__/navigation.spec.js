import { mount } from 'vue-test-utils'
import { createRenderer } from 'vue-server-renderer'
import Navigation from '@/components/navigation.vue'

describe('navigation', () => {
  it('has same HTML structure', () => {
    const mocks = {
      $route: {name: 'Form'}}
    const renderer = createRenderer()
    const wrapper = mount(Navigation, {mocks})
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })
})
