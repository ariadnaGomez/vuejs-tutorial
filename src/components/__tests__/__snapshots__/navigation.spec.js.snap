// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigation has same HTML structure 1`] = `
<el-menu router="true" default-active="3" mode="horizontal" background-color="#545c64" text-color="#fff" active-text-color="#ffd04b" class="menu">
    <el-menu-item route="[object Object]" index="1">
        Posts
    </el-menu-item>
    <el-menu-item route="[object Object]" index="2">
        Posts activos
    </el-menu-item>
    <el-menu-item route="[object Object]" index="3">
        Form
    </el-menu-item>
</el-menu>
`;
